src/main/java/seedu/socket/ui/HelpWindow.java

```diff
@@ -28,13 +28,24 @@ public class HelpWindow extends UiPart<Stage> {
             + "3. Editing a person : edit\n"
             + "Format: edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] "
             + "[g/GITHUBPROFILE] [l/LANGUAGE] [t/TAG]…\n\n"
-            + "4. Locating persons by name or tag: find\n"
+            + "4. Removing a person's attribute : remove\n"
+            + "Format: remove INDEX [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] \"\n"
+            + "[g/GITHUBPROFILE] [l/LANGUAGE] [t/TAG]…\n\n"
+            + "5. Clearing groups of person : clear\n"
+            + "Format: clear [t/TAG]...\n\n"
+            + "6. Locating persons by name or tag: find\n"
             + "Format: find KEYWORD [MORE_KEYWORDS]… [t/TAG]…\n\n"
```

Comment:
Ah I've forgotten to update the find command here... I'll update on my end after I merge.